### ISSUE TITLE
Failed to resolve: dev.skomlach:biometric:1.0.86

### ISSUE THREAD

**Issue by xvonabur** [Salat-Cx65/AdvancedBiometricPromptCompat#101]
Cannot install any version higher than 1.0.75.
Which repo do I need to use in gradle config?

**Comment by Salat-Cx65** [comment]
Perhaps some issue on bintray side.
I schedule [migration](https://github.com/Salat-Cx65/AdvancedBiometricPromptCompat/issues/82), but the issue not closed yet.

As a hotfix - you can try the next config
 1) Top  Gradle
`maven { url  "https://dl.bintray.com/salat-cx65/Maven" }`

2) App Gradle
`
implementation 'dev.skomlach:biometric:1.0.81'`


This ticket will be closed later after migration from JFrog

**Comment by xvonabur** [comment]
@Salat-Cx65 thanks! Unfortunately your solution doesn't work. I'm receiving 403 error:
```
Could not GET 'https://dl.bintray.com/salat-cx65/Maven/dev/skomlach/biometric/1.0.81/biometric-1.0.81.pom'. Received status code 403 from server: Forbidden

```

**Comment by Salat-Cx65** [comment]
@xvonabur library is now available again with version 1.0.90 on MavenCentral
